From 4b66206f0dd5985ab7068e91a9584e37ec28649c Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Tue, 26 Aug 2014 12:49:28 -0400
Subject: [tools] cpufreq: Make linux-pm@vger.kernel.org official mailing list

Message-id: <1409057382-26432-18-git-send-email-prarit@redhat.com>
Patchwork-id: 89105
O-Subject: [RHEL7.1 PATCH BZ 1133893 17/31] cpufreq: Make linux-pm@vger.kernel.org official mailing list
Bugzilla: 1133893
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1133893

commit dec102aa9ac112d66133314815d20233c96ad749
Author: Viresh Kumar <viresh.kumar@linaro.org>
Date:   Tue Apr 22 10:42:05 2014 +0530

    cpufreq: Make linux-pm@vger.kernel.org official mailing list

    There has been confusion all the time about which mailing list to follow
    for cpufreq activities, linux-pm@vger.kernel.org or cpufreq@vger.kernel.org.

    Since patches sent to cpufreq@vger.kernel.org don't go to Patchwork
    which is a maintenance workflow problem, make linux-pm@vger.kernel.org
    the official mailing list for cpufreq stuff and remove all references
    of cpufreq@vger.kernel.org from kernel source.

    Later, we can request that the list be dropped entirely.

    Signed-off-by: Viresh Kumar <viresh.kumar@linaro.org>
    [rjw: Changelog]
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Dean Nelson <dnelson@redhat.com>
Cc: Tony Camuso <tcamuso@redhat.com>
Cc: Lenny Szubowicz <lszubowi@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Documentation/ABI/testing/sysfs-devices-system-cpu b/Documentation/ABI/testing/sysfs-devices-system-cpu
index 468e4d4..8ba715d 100644
--- a/Documentation/ABI/testing/sysfs-devices-system-cpu
+++ b/Documentation/ABI/testing/sysfs-devices-system-cpu
@@ -128,7 +128,7 @@ Description: Discover cpuidle policy and mechanism
 
 What:  /sys/devices/system/cpu/cpu#/cpufreq/*
 Date:  pre-git history
-Contact: cpufreq@vger.kernel.org
+Contact: linux-pm@vger.kernel.org
 Description: Discover and change clock speed of CPUs
 
   Clock scaling allows you to change the clock speed of the
@@ -146,7 +146,7 @@ Description: Discover and change clock speed of CPUs
 
 What:  /sys/devices/system/cpu/cpu#/cpufreq/freqdomain_cpus
 Date:  June 2013
-Contact: cpufreq@vger.kernel.org
+Contact: linux-pm@vger.kernel.org
 Description: Discover CPUs in the same CPU frequency coordination domain
 
   freqdomain_cpus is the list of CPUs (online+offline) that share
diff --git a/Documentation/cpu-freq/index.txt b/Documentation/cpu-freq/index.txt
index 3d0b915..dc024ab 100644
--- a/Documentation/cpu-freq/index.txt
+++ b/Documentation/cpu-freq/index.txt
@@ -35,8 +35,8 @@ Mailing List
 ------------
 There is a CPU frequency changing CVS commit and general list where
 you can report bugs, problems or submit patches. To post a message,
-send an email to cpufreq@vger.kernel.org, to subscribe go to
-http://vger.kernel.org/vger-lists.html#cpufreq and follow the
+send an email to linux-pm@vger.kernel.org, to subscribe go to
+http://vger.kernel.org/vger-lists.html#linux-pm and follow the
 instructions there.
 
 Links
diff --git a/MAINTAINERS b/MAINTAINERS
index 0f2c923..5ce636f 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -2224,7 +2224,6 @@ F: drivers/net/ethernet/ti/cpmac.c
 CPU FREQUENCY DRIVERS
 M: Rafael J. Wysocki <rjw@sisk.pl>
 M: Viresh Kumar <viresh.kumar@linaro.org>
-L: cpufreq@vger.kernel.org
 L: linux-pm@vger.kernel.org
 S: Maintained
 T: git git://git.kernel.org/pub/scm/linux/kernel/git/rafael/linux-pm.git
@@ -2235,7 +2234,6 @@ F: include/linux/cpufreq.h
 CPU FREQUENCY DRIVERS - ARM BIG LITTLE
 M: Viresh Kumar <viresh.kumar@linaro.org>
 M: Sudeep KarkadaNagesha <sudeep.karkadanagesha@arm.com>
-L: cpufreq@vger.kernel.org
 L: linux-pm@vger.kernel.org
 W: http://www.arm.com/products/processors/technologies/biglittleprocessing.php
 S: Maintained
diff --git a/drivers/cpufreq/speedstep-centrino.c b/drivers/cpufreq/speedstep-centrino.c
index f897d51..b383213 100644
--- a/drivers/cpufreq/speedstep-centrino.c
+++ b/drivers/cpufreq/speedstep-centrino.c
@@ -28,7 +28,7 @@
 #include <asm/cpu_device_id.h>
 
 #define PFX  "speedstep-centrino: "
-#define MAINTAINER "cpufreq@vger.kernel.org"
+#define MAINTAINER "linux-pm@vger.kernel.org"
 
 #define INTEL_MSR_RANGE (0xffff)
 
diff --git a/tools/power/cpupower/Makefile b/tools/power/cpupower/Makefile
index cbfec92..3651db7 100644
--- a/tools/power/cpupower/Makefile
+++ b/tools/power/cpupower/Makefile
@@ -62,7 +62,7 @@ LIB_MAJ=   0.0.0
 LIB_MIN=   0
 
 PACKAGE =   cpupower
-PACKAGE_BUGREPORT =  cpufreq@vger.kernel.org
+PACKAGE_BUGREPORT =  linux-pm@vger.kernel.org
 LANGUAGES =    de fr it cs pt
 
 
diff --git a/tools/power/cpupower/debug/kernel/cpufreq-test_tsc.c b/tools/power/cpupower/debug/kernel/cpufreq-test_tsc.c
index 66cace6..34bdeb1 100644
--- a/tools/power/cpupower/debug/kernel/cpufreq-test_tsc.c
+++ b/tools/power/cpupower/debug/kernel/cpufreq-test_tsc.c
@@ -18,7 +18,7 @@
  * 5.) if the third value, "diff_pmtmr", changes between 2. and 4., the
  *     TSC-based delay routine on the Linux kernel does not correctly
  *     handle the cpufreq transition. Please report this to
- *     cpufreq@vger.kernel.org
+ *     linux-pm@vger.kernel.org
  */
 
 #include <linux/kernel.h>
-- 
1.7.1